deploy/gvisor/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:18.04
```

Comment:
do we really need ubuntu or can we get away with something smaller like alpine? or something distroless maybe?

Reply:
I initially tested other versions of ubuntu which didn't work (14.04 and 16.04), because I was getting errors with mounting directories and running `systemctl`.  The [nvidia](https://github.com/GoogleCloudPlatform/container-engine-accelerators/blob/master/nvidia-driver-installer/minikube/Dockerfile) addon uses the same base image, which is why I tried it initially, and it seems to work. 

I definitely think it would be worth testing a smaller base image, but right now I'm not sure if they would work.